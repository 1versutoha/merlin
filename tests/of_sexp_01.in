["find", "use", "sexplib"]
["tell", "struct", "type t = int with of_sexp\n"]
["tell", "end", null]
["errors"]
["dump", "env"]
