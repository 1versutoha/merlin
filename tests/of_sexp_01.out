["return",true]
["return",false]
["return",true]
["return",[]]
["return",[["File \"\", line 1, characters 5-13","type t = int"],["File \"\", line 1, characters 18-25","val t_of_sexp : Sexplib.Sexp.t -> t"]]]
